week-3-nestjs-back-end/w03e01-modular-app/food-delivery/src/auth/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const jwtConstants = {
+  secret: 'secretKey',
```

Comment:
'secretKey' musi być w `.env` bo inaczej jest "jawny" w kodzie na repo.


Zobacz, jak wrzucisz `secretKey` do .env - to żaden developer nawet NIE WIE, jaki `secret` jest ustawiony na prodzie i wiedzieć nie musi.

Wiedzą to tylko osoby odpowiedzialne za devOps.
Jednak - co ważniejsze, jeśli kod repo gdzieś "wycieknie" to mamy dalej bezpieczny secret.